tests/Avalonia.Base.UnitTests/AvaloniaObjectTests_Metadata.cs

```diff
@@ -67,7 +67,7 @@ private class Class1 : AvaloniaObject
             public static readonly DirectProperty<Class1, string> DirectProperty =
                 AvaloniaProperty.RegisterDirect<Class1, string>("Styled", o => o.Direct, unsetValue: "foo");
 
-            private string _direct;
+            private string _direct = default;
```

Comment:
Change not necessary - the default value of a reference type field is already `default`.

Reply:
AvaloniaObjectTests_Metadata.cs(70,28): warning CS0649: Field 'AvaloniaObjectTests_Metadata.Class1._direct' is never assigned to